client/src/components/ViewModeToggle/ViewModeToggle.scss

```diff
@@ -0,0 +1,67 @@
+.viewModeToggle {
+  margin-left: auto;
+
+  .viewMode-selected.btn {
+    &,
+    &:hover,
+    &:before,
+    &.highlighted,
+    &:focus {
+      background: $dropdown-link-active-bg;
+      color: $component-active-color;
+      border: 0;
+    }
+  }
+
+  #splitModeButton {
+    &:disabled {
+      background: red;
+      pointer-events: all;
+    }
+  }
+}
+
+.popover-container:before {
+  content: none;
+}
+
+.disabled-tooltip {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 37px;
+  background: rgba(255, 255, 255, 0);
+
+  &::before {
+    content: "Screen size too small";
```

Comment:
Is there any way of adding this with javascript or PHP? We can't make this string translatable if it's in CSS

Reply:
Rather than having a `:before` pseudo element, you could have a span styled the same and injected to the `.disabled-tooltip` div element, this will enable translatable strings and not require any hacks.

If you want the hacked version, use the `attr()` css function and pick an arbitrary attribute to get the translation string from.